README.md

```diff
@@ -136,3 +136,14 @@ and look up the external ip (a.b.c.d5) of the instance you just created (if you
 Note the phoenix webserver is running on port 4000
 Go to http://a.b.c.d:4000/
 Note it is http not https
+
+### Generating SBOM file
```

Comment:
update `generating SBOM file instruction ` to the format shared with you earlier